src/sentry/static/sentry/app/data/forms/projectGeneralSettings.tsx

```diff
@@ -95,14 +96,17 @@ export const fields = {
       "Automatically resolve an issue if it hasn't been seen for this amount of time"
     ),
     formatLabel: val => {
-      val = parseInt(val, 10);
+      val = Number(val);
```

Comment:
This will behave a bit different if the input value is float shaped.

Reply:
This should be OK